site/tests/app/controllers/admin/PlagiarismControllerTester.php

```diff
@@ -0,0 +1,34 @@
+<?php namespace tests\app\controllers\admin;
+
+use app\libraries\Core;
+use tests\BaseUnitTest;
+use app\controllers\admin\PlagiarismController;
+
+class PlagiarismControllerTester extends BaseUnitTest
+{
+   /** @var PlagiarismController */
+    protected $controller;
+
+    /** @var Core */
+    protected $core;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->core = $this->createMockCore([], [], [
+            'getGradeableConfig'  => [],
+        ]);
+
+        $this->controller = new PlagiarismController($this->core);
+    }
+
+    /** @test */
+    public function testQueriesMadeInPlagiarismView()
+    {
+        $response = $this->controller->editPlagiarismSavedConfig();
```

Comment:
No need to catch the return output of void. It doesn't return anything, so no need to capture the output.